app/kumactl/data/install/k8s/metrics/grafana/kuma-dataplane.json

```diff
@@ -1961,6 +2060,7 @@
       "steppedLine": false,
       "targets": [
         {
+          "datasource": "Prometheus",
```

Comment:
this capital case change is strange no?

Reply:
This entry is also line 2019 which is strange

Reply:
It is strange, but it is what Grafana produced, so I decided I won't argue with it. But I can revert it if you want

Reply:
No it's fine if it's coming from Grafana :)